src/platform/pic32/uart.cpp

```diff
@@ -163,7 +181,32 @@ bool openxc::interface::uart::connected(UartDevice* device) {
             status = false;
             break;
     }
-    return status;
 
 #endif
+
+	// on apple devices, it takes some time for flow control to propagate to the device
+	// during that time, if we transmit, we crash the connection
+	// simple solution: rising delay for device connected status
+	if(last_status == false && status == true)
```

Comment:
The braces here don't match the style of the rest of the code - start on the same line as the conditional.